nuntium/templates/nuntium/profiles/message_detail.html

```diff
@@ -27,14 +27,19 @@
           </dd>
 	  <dt>{% trans "Confirmed" %}</dt>
 	  <dd><i class="fa {% if message.confirmated %}fa-check{% else %}fa-times{% endif %}"></i></dd>
+
 	  {% if message.writeitinstance.config.moderation_needed_in_all_messages %}
 	  	<dt>{% trans "Moderated" %}</dt>
-        <dd><i class="fa {% if message.moderated %}fa-check{% else %}fa-times{% endif %}"></i></dd>
-      {% endif %}
-      {% if message.public %}
-	  <dt>{% trans "Public" %}</dt>
-	  <dd><i class="fa {% if message.public %}fa-check{% else %}fa-times{% endif %}"></i></dd>
-      {% endif %}
+        {% if message.moderated %}
```

Comment:
The indentation on this whole template is all over the place. We have a mixture of 2, 4, 6, and 8-space indentation at various places!